radiation/python/radlw/cldprop_gt4py.py

```diff
@@ -17,7 +17,7 @@
     __externals__,
 )
 
-sys.path.insert(0, "/Users/AndrewP/Documents/work/physics_standalone/radiation/python")
+sys.path.insert(0, "..")
```

Comment:
yay! I think the sys.path.insert is not the only way to make packages on that level importable, but this works and removes the hard-coding, so awesome!